projects/plugins/jetpack/src/class-deprecate.php

```diff
@@ -119,7 +183,11 @@ private function render_notice( $id, $text, $params = array() ) {
 			$params['additional_classes'][] = 'jetpack-deprecate-dismissible';
 		}
 
-		wp_admin_notice( $text, $params );
+		printf(
```

Comment:
Do we need the `dismissible` logic above? Actually, it looks like we no longer need lines 170-185, no?

Reply:
Good call. In fact we don't need `$params['id'] = $id;` either, nor to pass the `$params` array into the function. If anyone finds they need additional control for any reason, some of the params and conditionals could be added back later. Changed in de5ae51d520b101582573b659068edb117df38d4